.github/workflows/release.yml

```diff
@@ -1,6 +1,12 @@
 on: 
   release:
     types: [created]
```

Comment:
How about adding `updated`?

Reply:
`release` trigger: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release

- published
- unpublished
- created
- edited
- deleted
- prereleased
- released

`edited` is triggered only when the description is updated so that's not appropriate I think.

Looks like [`published`](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release) is better than `created`.

Reply:
> I'm not sure why but the release workflow for [0.27.0](https://github.com/giginet/Scipio/releases/tag/0.27.0) didn't ran

That was because I created the release as draft initially.

https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=created#release

> A draft was saved, or a release or pre-release was published without previously being saved as a draft.